eks/outputs.tf

```diff
@@ -1,11 +1,14 @@
 output "id" {
-  value = module.thanos.id
+  description = "ID to pass other modules in order to refer to this module as a dependency. It takes the ID that comes from the main module and passes it along to the code that called this variant in the first place."
+  value       = module.thanos.id
 }
 
 output "metrics_archives" {
-  value = local.metrics_archives
+  description = "Bucket configuration in an HCL structure that will be used as an output to pass on to the module `kube-prometheus-stack` to activate and then configure `thanos-sidecar`."
+  value       = local.metrics_archives
 }
 
 output "thanos_enabled" {
```

Comment:
I dont think it is necesary to repeat "thanos", instead I would just output "enabled"
Referencing the output as `module.thanos.enabled` seems cleaner than `module.thanos.thanos_enabled`
I know this is not the scope of your PR but maybe we can already fix this?

Reply:
We did similar changes in the `kube-prometheus-stack`. In that module, since there are multiple components, we had to distinguish them by their name, like `prometheus_enabled` or `grafana_enabled`. Although it is redundant in this module's case because we would have `module.thanos.thanos_enabled`, I thought it would be better to follow the same naming convention of the other module. 

And the other issue I see with naming the output simply `enabled` is that in the event that the user calls the module something else rather than `thanos` on their DevOps Stack module's instantiations, then it would be difficult to know what this output is referencing, even though this change on the user's part would not be recommended...